src/web/mjs/connectors/ScanManga.mjs

```diff
@@ -72,6 +72,37 @@ export default class ScanManga extends Connector {
         }));
     }
 
+    async _getPages(chapter) {
+        let uri = new URL(chapter.id, this.url);
+        uri.searchParams.set('style', 'list');
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '.aLN');
+        return data.length > 0 ? this._getPagesNovel(request) : this._getPagesManga(chapter);
+    }
+
+    async _getPagesNovel(request) {
+        let script = `
+            new Promise(resolve => {
```

Comment:
When using reject, it should be defined in the callback for the promise as parameter
`new Promise((resolve, reject) => { ... });`